lte/gateway/c/session_manager/PipelinedClient.cpp

```diff
@@ -21,42 +21,30 @@ using grpc::Status;
 
 namespace {  // anonymous
 using std::experimental::optional;
-
-void call_back_void_upf(grpc::Status, magma::UpfRes response) {
-  MLOG(MDEBUG) << "Handle UPF response reached";  // ToDo
-}
-std::function<void(grpc::Status, magma::UpfRes)> callback = call_back_void_upf;
-
 // Preparation of Set Session request to UPF
 magma::SessionSet create_session_set_req(
     magma::SessionState::SessionInfo info) {
   magma::SessionSet req;
   magma::lte::Fsm_state_FsmState state         = info.state;
-  std::string seid                             = info.sess_id;
+  std::string seid                             = info.subscriber_id;
```

Comment:
change the variable name from seid to sid for subscriber_id

Reply:
changed, resolved.